Source/WebCore/html/HTMLTableCellElement.cpp

```diff
@@ -59,8 +59,11 @@ unsigned HTMLTableCellElement::colSpan() const
 
 unsigned HTMLTableCellElement::rowSpan() const
 {
-    // FIXME: a rowSpan equal to 0 should be allowed, and mean that the cell is to span all the remaining rows in the row group.
-    return std::max(1u, rowSpanForBindings());
+    unsigned rowSpanValue = rowSpanForBindings();
+    // when rowspan=0, the specs says it should apply to the full rows.
```

Comment:
Could you please be a bit more specific about the 'specs' here? (I know you linked to it in the commit message, but I think this comment may read better with a tiny bit more context)

Reply:
@alanbaradlay yes sure. Would that work?

```suggestion
    // when rowspan=0, the specs says it should apply to the full rows.
    // In https://html.spec.whatwg.org/multipage/tables.html#attr-tdth-rowspan
    // > For this attribute, the value zero means that the cell is
    // > to span all the remaining rows in the row group.
```

Reply:
done. (sorry for the delay I was on sick leave)